spec/models/commodity_spec.rb

```diff
@@ -18,6 +18,10 @@
       ]
     end
 
+    before do
```

Comment:
Do we want to validate the message sending to this guy or can we remove it?

Reply:
I think we're good here.